org.musicbrainz.Picard.appdata.xml

```diff
@@ -51,6 +51,7 @@
   </content_rating>
 
   <releases>
+    <release date="2018-08-10" version="2.0.3"/>
```

Comment:
Eventually, we should try and have some release notes here.

Maybe for future versions. :slightly_smiling_face: